src/riak_kv_ts_util.erl

```diff
@@ -534,28 +534,38 @@ key_element_to_string(V) -> lists:flatten(io_lib:format("~p", [V])).
 
 %%
 %% This one quotes values which should be quoted for assignment
-element_to_quoted_string(V) when is_binary(V) -> varchar_quotes(V);
-element_to_quoted_string(V) when is_float(V) -> mochinum:digits(V);
-element_to_quoted_string(V) -> lists:flatten(io_lib:format("~p", [V])).
+element_to_quoted_string(V, blob) -> hex_as_string(hexlify(V));
```

Comment:
The `mochihex` module is used in riak_ql. I used that instead of rolling my own so I didn't have to add unit tests to prove correctness.